internal/components/setup/common.go

```diff
@@ -64,6 +64,13 @@ func RunCommandsAndWait(runs []config.Run, timeout time.Duration) error {
 	return nil
 }
 
+// parseRunCommand parses command lines to individual commands.
+func parseRunCommand(command string) (commands []string) {
+	replacedCommand := strings.ReplaceAll(command, "\\\n", " ")
+	commands = strings.Split(replacedCommand, "\n")
+	return commands
+}
+
```

Comment:
Can we simply run the `command` as a whole instead of parsing like this?

Reply:
I think running commands separately can lets the user know what command is wrong when something exception happen. What do u think?

Reply:
okay, i will try it out